models/issues/issue.go

```diff
@@ -1344,14 +1348,23 @@ func (opts *IssuesOptions) setupSessionNoLimit(sess *xorm.Session) {
 	}
 
 	if opts.LabelIDs != nil {
+		var noLabelsIDs []int64
```

Comment:
They are excluded Labels, not no Label. So you still missed real No Label issues.

Reply:
this is done by set all lables who you dould select as exclude ...
... so it works

the question is if se should use a keyword instead

Reply:
If there are hundreds labels in this repository, it's difficult to select labels from the UI?